lib/vm/src/store.rs

```diff
@@ -104,6 +105,19 @@ impl StoreObjects {
         }
     }
 
+    /// Return an immutable iterator over all globals
+    pub fn iter_globals(&self) -> Iter<VMGlobal> {
+        self.globals.iter()
+    }
+
+    /// Set a global, at index idx. Will panic if idx is out of range
+    pub fn set_global(&self, idx: usize, val: u128) {
+        assert!(idx >= self.globals.len());
+        unsafe {
+            self.globals[idx].vmglobal().as_mut().val.u128 = val;
```

Comment:
Since this circumvents the type checks that we get with `Global`, maybe we should call the function `set_global_unchecked`, and make it clear in the function comment that the caller is responsible for writing a correct value for the globals type.